### ISSUE TITLE
Create The About And Contact Page

### ISSUE THREAD

**Issue by ayush-Shandilya** [ayush-Shandilya/STILLLEARN#3]

**Comment by charvibhargava** [comment]
I can add this. Please assign this to me.

**Comment by ayush-Shandilya** [comment]
Yeah, Simply read the docs and submit a pull request. After evaluating it, I will
merge it.

**Comment by sidhartha2002** [comment]
I have fixed #3 issue by adding about and contact page . 
Already made a pull request kindly verify it.

**Comment by charvibhargava** [comment]
Should I proceed with adding a contact and about page? @ayush-Shandilya

**Comment by ayush-Shandilya** [comment]
@charvibhargava The pages are already added, you could proceed with improving the UI.
